packages/jest-haste-map/src/HasteFS.ts

```diff
@@ -33,7 +33,11 @@ export default class HasteFS {
 
   getDependencies(file: Config.Path): Array<string> | null {
     const fileMetadata = this._getFileData(file);
-    return (fileMetadata && fileMetadata[H.DEPENDENCIES]) || null;
+    return (
+      (fileMetadata &&
+        fileMetadata[H.DEPENDENCIES].split(H.DEPENDENCY_DELIM)) ||
```

Comment:
We should handle the case where the files doesn't have any dependencies. In this case we'd be doing `''.split('\0')` which is `['']`. That might cause issues.

Reply:
Thought I had handled this, great catch!